doc/index.rst

```diff
@@ -22,3 +22,44 @@ c-lightning Documentation
    Coding Style Guideline <STYLE.md>
    MAKING-RELEASES.md
    CHANGELOG.md
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Manpages
+
+   doc/lightning-cli.1.md
```

Comment:
These paths are relative to the `index.rst` file, therefore should not have the `doc/` path.